control/controlclient/map_test.go

```diff
@@ -321,6 +427,12 @@ func formatNodes(nodes []*tailcfg.Node) string {
 		if n.LastSeen != nil {
 			extra += fmt.Sprintf(", lastSeen=%v", n.LastSeen.Unix())
 		}
+		if n.Key != (key.NodePublic{}) {
+			extra += fmt.Sprintf(", key=%v", n.Key.String())
```

Comment:
phew, this += string stuff is only in tests :)

extra silly that it's being done in a func that's already using strings.Builder 😂

Reply:
Heh, fixed that up while I was here 😃